### PR TITLE
Better support of HTMLSelectElement

### PR DESCRIPTION
**Pull request by paed01**

### CODE REVIEW

File: lib/HTMLOptionsCollection.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        let match;
+        const list = this[kLiveList];
+        for (let i = list.length - 1; i > -1; i--) {
+          if (i === value) match = true;
+          list[i].$elm.prop("selected", i === value);
+        }
+        if (!match) return this[kSelectedIndex] = -1;
+        return value;
+      }
+    });
```
[the remark this answers is not in the document]
Comment (paed01) [reply lib/HTMLOptionsCollection.js:33]: The proxy will not be able to pick them up if they are not enumerable.

[the remark this answers is not in the document]
Comment (paed01) [reply lib/HTMLOptionsCollection.js:33]: and class props, AFAIK, not enumerable

[the remark this answers is not in the document]
Comment (jonaswalden) [reply lib/HTMLOptionsCollection.js:33]: ☑️

**Review by jonaswalden — APPROVED** [review]
good stuff!

**Pull request merged**